README.md

```diff
@@ -36,7 +36,57 @@ See below the list of supported models:
 - [Default Value (CSV)](./readme/default_value.md)
 - [Language (XML)](./readme/language.md)
 
+### Specific model behavior parameters:
 
+Below is the list of supported optional parameters, that can change behavior when handling model files:
+
+#### field mapping:
+Map field name in Odoo model with column name in CSV file, see example [here](./odoo_initializer/models/orders_loader.py#L6-L14)
+
+```
+field_mapping = {
+        "lst_price": "odoo_price",
+        "product_variant_ids/categ_id/id": "odoo_category",
+        "type": "odoo_type",
+        "name": "Short name:en",
+        "product_variant_ids/uuid": "Uuid",
+        "id": "odoo_id",
+        "description": "Data class",
+    }
+```
+Field mapping is a JSON object where keys are Odoo field names (eg: "lst_price") and values are CSV column names (eg: "odoo_price")
+
```

Comment:
Could you explain what is this for?
In that case, it allows Odoo initializer to process Concept files from OpenMRS Initializer compatible files.